# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/linux/hotfix-dreamcast-dma.patch.superh
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- drivers/video/fbdev/pvr2fb.c.vanilla	2025-07-20 10:27:23.073616747 +0200
+++ drivers/video/fbdev/pvr2fb.c	2025-07-20 10:28:29.668570991 +0200
@@ -192,7 +192,7 @@
 
 #ifdef CONFIG_PVR2_DMA
 static unsigned int shdma = PVR2_CASCADE_CHAN;
-static unsigned int pvr2dma = ONCHIP_NR_DMA_CHANNELS;
+static unsigned int pvr2dma = CONFIG_NR_ONCHIP_DMA_CHANNELS;
 #endif
 
 static struct fb_videomode pvr2_modedb[] = {
